.gitlab-ci.yml

```diff
@@ -711,7 +711,16 @@ library:ci-color:
   - plugin:ci-bignums
 
 library:ci-compcert:
+  stage: stage-3
   extends: .ci-template-flambda
+  needs:
+  - build:edge+flambda
+  - library:ci-flocq
+  - library:ci-menhir
+  dependencies:
+  - build:edge+flambda
+  - library:ci-flocq
+  - library:ci-menhir
```

Comment:
According to the info found by @SkySkimmer and leading to #13471, there is no need to keep these lines. Though if you prefer, this can be kept here and removed in #13471 when rebasing on top.

Reply:
done